app/subscriber/src/components/content-list/ContentList.tsx

```diff
@@ -146,6 +154,7 @@ export const ContentList: React.FC<IContentListProps> = ({
 
   return (
     <styled.ContentList className="content-list" scrollWithin={scrollWithin}>
+      <Loader visible={requests.some((r) => r.url === 'find-contents-with-elasticsearch')} />
```

Comment:
Now all places this component is used it can show a loader that is only over this section.